core/network-libp2p/Cargo.toml

```
@@ -26,4 +26,4 @@ unsigned-varint = { version = "0.2.1", features = ["codec"] }
 
 [dev-dependencies]
 assert_matches = "1.2"
-parity-bytes = "0.1"
+parity-bytes = { git = "https://github.com/paritytech/parity-common", rev = "616b40150ded71f57f650067fcbc5c99d7c343e6" }
```

It needs the `Cargo.locks` file to take effect. Also, to make it easier to update, we can set this line to:

```suggestion
parity-bytes = { git = "https://github.com/paritytech/parity-common" }
```

And then, only update it in `Cargo.locks` by:

```
cargo update -p parity-bytes --precise 616b40150ded71f57f650067fcbc5c99d7c343e6
```

---

I'm not sure, when I apply your patch I get troubles with different locations of `parity-common`:
```
[nix-shell:~/devel/substrate]$ cargo vendor
    Updating git repository `https://github.com/paritytech/parity-common`
error: failed to sync

Caused by:
  found duplicate version of package `parity-bytes v0.1.0` vendored from two sources:

        source 1: https://github.com/paritytech/parity-common#15499079
        source 2: https://github.com/paritytech/parity-common?rev=616b40150ded71f57f650067fcbc5c99d7c343e6#616b4015
```